layers/meta-balena-raspberrypi/recipes-bsp/rpi-eeprom/files/default-config.txt

```diff
@@ -2,3 +2,4 @@ BOOT_UART=0
 WAKE_ON_GPIO=1
 POWER_OFF_ON_HALT=0
 FREEZE_VERSION=0
+BALENA_HOST_CONFIG_gpu_mem=16
```

Comment:
@sradevski Isn't this the firmware configuration file as described in https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloader_config.md? How does a BALENA_HOST configuration variable gets applied here?

Reply:
@alexgg I'm not sure honestly, maybe you can point me to the right place to add this. What I want to do is add `BALENA_HOST_CONFIG_gpu_mem` as part of the `config.txt` file on the device, which the supervisor then picks up and sets. I might be misunderstanding something though, so some guidance would be helpful here.
See arch call item: https://app.frontapp.com/open/msg_exxdqrx

Reply:
@sradevski I would need to revisit the arch call to get the context, but I think the config.txt file you are looking for is not this one but the one modified in https://github.com/agherzan/meta-raspberrypi/blob/09a3c11696c75593f8e475da5dfb401016c6aaca/recipes-bsp/bootfiles/rpi-config_git.bb#L70, and then appended in https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

Reply:
@alexgg Yeah that's what I am looking for, thanks for helping out. The names are misleading for sure if you are not familiar with the repo for sure :)

Reply:
The `default-config.txt` file is used to modify the bootloader configuration at compilation time, and it does not finish up in the target as far as I know.

Reply:
@alexgg I'm quite confused about how default values are set. [here](https://github.com/agherzan/meta-raspberrypi/blob/09a3c11696c75593f8e475da5dfb401016c6aaca/docs/extra-build-config.md#gpu-memory) it says the default GPU_mem is 64MB, but I don't see that value set anywhere, where do we set that? Also, where does the`GPU_MEM` variable come from that is used [here](https://github.com/agherzan/meta-raspberrypi/blob/09a3c11696c75593f8e475da5dfb401016c6aaca/recipes-bsp/bootfiles/rpi-config_git.bb#L70), I guess we set it at build time? Does that mean we just need to set GPU_MEM in our build scripts to have the default gpu mem value set to 16?

Reply:
@sradevski looks like GPU_MEM is set in https://github.com/balena-os/balena-raspberrypi/blob/654e7e054253111d349be15c77d17325bcbb95d3/layers/meta-balena-raspberrypi/conf/samples/local.conf.sample#L21 , so currently all boards already have the value 16 set at build time. At least all raspberrypi boards that install rpi-config_git.bb.

Reply:
@acostach I'm super confused now, in the arch call I was led to believe that it isn't. Anyway, thanks for pointing it out, I'll close the PR now